pebblo/app/config/config.py

```diff
@@ -67,6 +75,7 @@ def load_config(path: str) -> Tuple[dict, Config]:
             ),
             logging=LoggingConfig(),
             classifier=ClassifierConfig(anonymizeSnippets=False),
+            storage=StorageConfig(type="file", db=None),
```

Comment:
Add a comment here indicating this will switch to `db` and `sqlite` in the next release. In fact, see if you can make this default emit a deprecation warning in the console / log.